### ISSUE TITLE
Unable to upload groups form FilesToxMatters.js

### ISSUE THREAD

**Issue by Abdelbary** [brannonvann/xmtoolbox-quick-start#7]
when using .\FilesToxMatters.js to upload groups I get this error 
```
Error: {"code":400,"message":"Either id or target name is required","reason":"Bad Request","subcode":""}
    {
  statusCode: 400,
  data: '{"code":400,"message":"Either id or target name is required","reason":"Bad Request","subcode":""}',
  options: Url {
    protocol: 'https:',
    slashes: true,
    auth: null,
    host: 'vodafone-np.hosted.xmatters.com',
    port: null,
    hostname: 'vodafone-np.hosted.xmatters.com',
    hash: null,
    search: null,
    query: null,
    pathname: '/api/xm/1/groups',
    path: '/api/xm/1/groups',
    href: 'https://vodafone-np.hosted.xmatters.com/api/xm/1/groups',
    method: 'POST',
    headers: {
    
    },
    agent: undefined,
    timeout: 5000,
    data: '{"description":"with userUploadToImport or after using the groupTransform sync option."}'
  }
}
```

**Comment by brannonvann** [comment]
Hi - 
The error you provided comes from xMatters directly and describes the issue. It appears you have not provided a group name (targetName) or id. Can you please share an example of what you are attempting to upload and whether or not you expect the group to be created or updated?  From this message above it seems like it's attempting to create a new group. Is this your expectation? If so every group needs a name.

Also, please change the password for this user or API key as soon as possible. You have posted the authentication (headers: Authorization) credentials in this ticket. Please redact these in the future for your protection.

Regards,
Brannon

**Comment by brannonvann** [comment]
@Abdelbary Is this still an issue?

**Comment by Abdelbary** [comment]
yes, actually the problem was in this line here in FileToXmatter.js
```
  const groupFields = ['name', 'description'];
```
where the first-filed was called name I changed it to be targetName so that it can be added to the json payload as targetName
```
  const groupFields = ['targetName', 'description'];
```

**Comment by brannonvann** [comment]
Thank you for reporting this. It has been fixed in commit 31a665f.

**Comment by Abdelbary** [comment]
Couldn't you accept my pull request instead, I really wanted to contribute to an open-source project :( .
Anyways the package is really powerful and generic, and I am learning javascript and nodeJS just to fully understand the it :D
